src/Orleans.Runtime/Services/GrainService.cs

```diff
@@ -17,9 +17,8 @@ public abstract class GrainService : SystemTarget, IRingRangeListener, IGrainSer
         private readonly IConsistentRingProvider ring;
         private readonly string typeName;
         private GrainServiceStatus status;
-
-        /// <summary>Logger instance to be used by grain service subclasses</summary>
-        protected Logger Logger { get; }
+        
+        private ILogger Logger;
```

Comment:
Is this accessibility change by design? In the past it was so that derived classes (app level) can use it

Reply:
Maybe it's fine, as you mentioned in person, now it's very easy to get a logger from DI. In the past that was not possible